wls-gui-admintool/tests/components/wahltag/BaseDialogWahltagOverrideWahlterminConfirmation.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { getSnapshotFilename } from "@tests/utils/testutils.ts";
+import { mount, VueWrapper } from "@vue/test-utils";
+import { afterEach, beforeEach, describe, expect, it } from "vitest";
+import { nextTick } from "vue";
+import { VTextField } from "vuetify/components";
+
+import BaseButtonCancel from "@/components/common/BaseButtonCancel.vue";
+import BaseButtonConfirm from "@/components/common/BaseButtonConfirm.vue";
+import BaseDialogWahltagOverrideWahlterminConfirmation from "@/components/wahltag/BaseDialogWahltagOverrideWahlterminConfirmation.vue";
+import vuetify from "@/plugins/vuetify.ts";
+
+describe("BaseDialogWahltagOverrideWahlterminConfirmation.vue", () => {
+  let wrapper: VueWrapper<
+    InstanceType<typeof BaseDialogWahltagOverrideWahlterminConfirmation>
+  >;
+
+  beforeEach(() => {
+    wrapper = setupWrapper();
+  });
+
+  afterEach(() => {
+    cleanUp(wrapper);
+  });
+
+  describe("visual logic", () => {
+    it("should_nothingVisible_when_mountedUnchanged", async (context) => {
+      await expect(document.body.innerHTML).toMatchFileSnapshot(
+        getSnapshotFilename(context)
+      );
+    });
+    it("should_renderDialog_when_showWasCalled", async (context) => {
+      wrapper.vm.show();
+
+      await wrapper.vm.$nextTick();
+
+      await expect(document.body.innerHTML).toMatchFileSnapshot(
+        getSnapshotFilename(context)
+      );
+    });
+
+    it("should_hideDialog_when_hideWasCalledAfterShow", async (context) => {
+      wrapper.vm.show();
+      await wrapper.vm.$nextTick();
+      wrapper.vm.hide();
+      await wrapper.vm.$nextTick();
+
+      await expect(document.body.innerHTML).toMatchFileSnapshot(
+        getSnapshotFilename(context)
+      );
+    });
+
+    it("should_renderValues_when_propertiesAreSet", async (context) => {
+      cleanUp(wrapper);
+
+      wrapper = setupWrapper({
+        requiredConfirmText: "requiredConfirmText",
+      });
+
+      wrapper.vm.show();
+
+      await wrapper.vm.$nextTick();
+
+      await expect(document.body.innerHTML).toMatchFileSnapshot(
+        getSnapshotFilename(context)
+      );
+    });
+  });
+
+  describe("behavior logic", () => {
```

Comment:
```suggestion
  describe("behavioral logic", () => {
    describe("confirmDelete", () => {
```
nach der naming convention fehlt hier nochmal eine verschachtelung nach events

Reply:
Konstante verwenden

Reply:
Konstante eingeführt mit 0fae347b711eee296453f302668793a6193b84ca